xml/System.Runtime.InteropServices/DefaultParameterValueAttribute.xml

```diff
@@ -64,16 +64,20 @@
 ## Remarks  
  The <xref:System.Runtime.InteropServices.DefaultParameterValueAttribute> attribute allows you to specify a default parameter value in a language that does not otherwise support default parameters.  After you apply this attribute to your code, languages that support default parameters can use the specified value as a default parameter.  
   
- Note that the <xref:System.Runtime.InteropServices.DefaultParameterValueAttribute> does not add support for default parameters to languages that do not support this feature.  For example, if you use the <xref:System.Runtime.InteropServices.DefaultParameterValueAttribute> with a method written in C#, which does not support default parameters, you cannot use the default parameter when calling the method from C#.  You can use the default parameter only when calling the method in a language such as C++ that does support the feature.  
+ Note that the <xref:System.Runtime.InteropServices.DefaultParameterValueAttribute> does not add support for default parameters to languages that do not support this feature.  For example, if you use the <xref:System.Runtime.InteropServices.DefaultParameterValueAttribute> with a language that does not support default parameters, you cannot use the default parameter when calling the method in said language.  You can use the default parameter only when calling the method in a language such as C++ that does support the feature.  
   
  The <xref:System.Runtime.InteropServices.DefaultParameterValueAttribute> is particularly useful to specify default parameters for methods of a COM interop interface.  
   
    
   
 ## Examples  
- The following code example demonstrates how to apply the <xref:System.Runtime.InteropServices.DefaultParameterValueAttribute> attribute to a parameter of a method written in C#.  
+ The following code example demonstrates how to apply the <xref:System.Runtime.InteropServices.DefaultParameterValueAttribute> attribute to a parameter of a method written in C#.  The <xref:System.Runtime.InteropServices.OptionalAttribute> attribute is used to allow calling of the method without any arguments.  
   
- :::code language="csharp" source="~/snippets/csharp/System.Runtime.InteropServices/DefaultParameterValueAttribute/Overview/sample.cs" id="Snippet1":::  
+ :::code language="csharp" source="~/snippets/csharp/System.Runtime.InteropServices/DefaultParameterValueAttribute/Overview/sample1.cs" id="Snippet1":::  
+  
+ Usage of <xref:System.Runtime.InteropServices.DefaultParameterValueAttribute> and <xref:System.Runtime.InteropServices.OptionalAttribute> does have some differences as compared to normal C# default parameters.  
```

Comment:
```suggestion
There are some behavior differences between [C# optional arguments](/dotnet/csharp/programming-guide/classes-and-structs/named-and-optional-arguments#optional-arguments) and using <xref:System.Runtime.InteropServices.DefaultParameterValueAttribute> and <xref:System.Runtime.InteropServices.OptionalAttribute>. The following code snippet demonstrates these differences.
```

Reply:
I replaced the `and` between the 2 attr xrefs with `with`. Hope that is better (current one has too many `and`s).